### DISCUSSION TITLE
Loop feature in animation

### DISCUSSION THREAD

**Discussion by mcbeaker** [scanberg/viamd#45] [Q&A] [answered]
Hi! I am probably missing the toggle for this completely in the docs/paper. Is there a toggle for looping animations continuously? Also, in order to replay the sim I need hit stop first and then play/spacebar - is that an intended behavior?

**Comment by mathieulinares** [comment, marked as answer]
Nope there is not yet. But we agree that it should be implemented in the future. Regarding the pressing stop before playing again, it is an expected behavior at the moment but it should be improved so the user does not have to press the stop button to play the trajectory again.
